js/speakers.js

```diff
@@ -0,0 +1,56 @@
+const speakers = document.getElementById('speakers');
+const speakerData = [
+  {
+    name: 'Daniel Vera',
+    role: 'Bass, main vocals, lyrics composer',
+    description: 'Dasik co-founder. Music & lyrics composer. Bass player. Producer. Leader of Dasik Band project',
+    image: './images/DanielVera.jpeg',
+  },
+  {
+    name: 'David Vera',
+    role: 'Guitar, backing vocals, music composer',
+    description: 'Dasik co-founder. Music composer. Guitar and bass player. Main arranger for the songs. Playing guitar since 2003',
+    image: './images/DavidVera.jpeg',
+  },
+  {
+    name: 'Damian Betancourt',
+    role: 'External consultant and touring musician',
+    description: 'Producer and touring musician. Multi-instrumentalist. Engineer. Music consultant',
+    image: './images/DamianBetancourt.jpeg',
+  },
+  {
+    name: 'Xavier De La Torre',
+    role: 'Touring musician. Drummer',
+    description: 'Drummer and percusionist. Touring musician.',
+    image: './images/XavierDeLaTorre.jpeg',
+  },
+];
+
+function generateLi(index) {
+  const li = document.createElement('li');
+  const photo = document.createElement('div');
+  const info = document.createElement('div');
+  const h3 = document.createElement('h3');
+  const h4 = document.createElement('h4');
+  const p = document.createElement('p');
+
+  photo.classList.add('photo');
+  info.classList.add('info');
+
+  photo.innerHTML = `<div class="pattern"> </div><img src="${speakerData[index].image}" alt="${speakerData[index].name}"></div>`;
+  h3.innerText = speakerData[index].name;
+  h4.innerText = speakerData[index].role;
+  p.innerHTML = speakerData[index].description;
+
+  info.appendChild(h3);
+  info.appendChild(h4);
+  info.appendChild(p);
+
+  li.appendChild(photo);
+  li.appendChild(info);
+  return li;
```

Comment:
- [ ] Great job generating your speaker's section dynamically from JavaScript 💯 In the mobile view, only 2 speakers should be displayed together with a `more` button. Kindly rectify this issue.
- [ ] (OPTIONAL) Also when a user clicks the `more` button the rest of the speakers are then displayed, it would be great if you could implement this functionality so as to improve the UI of your site 👍 
- [ ] Kindly add the red horizontal line under `band collaborators` so as to stick to the original design.
- Your design:
![Annotation 2022-05-19 040711](https://user-images.githubusercontent.com/85113567/169196729-c41b8eda-edfb-462b-94f0-39125f1e38d5.jpg)
- Required design:
![Annotation 2022-05-19 040653](https://user-images.githubusercontent.com/85113567/169196764-62ca0d93-03e5-477c-bb49-da6ff99744db.jpg)